library/Icingadb/Widget/ObjectsStatistics.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/* Icinga DB Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget;
+
+use Icinga\Chart\Donut;
+
+use Icinga\Module\Icingadb\Model\RedundancyGroupParentStateSummary;
+use Icinga\Module\Icingadb\Widget\Detail\ObjectStatistics;
+use ipl\Html\Text;
+use ipl\Html\ValidHtml;
+use ipl\Html\HtmlString;
+
+/**
+ * Objects statistics
```

Comment:
Same here :sweat_smile: `DependencyNodeStatistics` and inside, the properties can be prefixed by `nodes_`